internal/logutils/logger.go

```diff
@@ -0,0 +1,21 @@
+package logutils
```

Comment:
> Avoid package names like base, util, or common

https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common

Reply:
Would `internal/logging` name be appropriate?